src/crimson/os/seastore/omap_manager/btree/omap_btree_node.h

```diff
@@ -109,6 +110,12 @@ struct OMapNode : LogicalChildNode {
   virtual uint32_t get_node_size() = 0;
 
   virtual ~OMapNode() = default;
+
+  bool is_btree_root() const { return root; }
+  void unset_root() { root = false; }
+  void set_root() { root = true; }
+private:
+  bool root = false;
```

Comment:
Is this leaking the changes between pending/unsubmitted transactions?

Reply:
Will fix this issue